internal/apps/admin/manager/cluster.go

```diff
@@ -74,15 +77,11 @@ func (am *AdminManager) ListCluster(ctx context.Context, req *http.Request, reso
 		}
 		newClusters = resp.Data
 	} else {
-		orgID, err = GetOrgID(req)
-		if err != nil {
-			return apierrors.ErrListCluster.InvalidParameter(err).ToResp(), nil
-		}
-
-		clusterRelation, err := am.bundle.GetOrgClusterRelationsByOrg(orgID)
+		orgResp, err := am.org.GetOrgClusterRelationsByOrg(apis.WithInternalClientContext(ctx, "admin"), &orgpb.GetOrgClusterRelationsByOrgRequest{OrgID: orgIDStr})
```

Comment:
use discover.SvcXXX for literal `admin`.

Reply:
done